libs/api/domains/consultation-portal/src/lib/cases/cases.resolver.ts

```diff
@@ -0,0 +1,40 @@
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql'
+import { CaseResultService } from './cases.service'
+import { CaseResult } from '../models/caseResult.model'
+import { CaseItemResult } from '../models/caseItemResult.model'
+import { AdviceResult } from '../models/adviceResult.model'
+
+@Resolver()
+export class CaseResultResolver {
```

Comment:
We should feature flag these resolvers since these resolvers will be deployed to production (in the next release-train) even though the backend isn't ready.

We do have a special [server-side feature flag system](https://docs.devland.is/technical-overview/devops/service-setup#server-side-feature-flags) which is designed to support "missing environment variables" in production but it's a bit more difficult to work with.

Since you're not dealing with "missing secrets" yet and have already configured a (test) backend url for all environments, it's probably easiest to use ConfigCat feature flags. Check out our [NestJS module and guard](https://docs.devland.is/libs/nest/feature-flags).